.pylintrc

```diff
@@ -184,7 +184,9 @@ disable=print-statement,
         W0612,
         W0613,
         W0621,
-        invalid-docstring-quote
+        W0707,
```

Comment:
As this list changes now, can you please add them to https://github.com/elastic/rally/issues/838?